### PR TITLE
Firebase config

### PR DESCRIPTION
**Pull request by udaraJ08**
[ITP-135] firebase configured. And fetched the data from firestore and displayed

[ITP-135]: https://fcodelabs.atlassian.net/browse/ITP-135

### CODE REVIEW

File: src/pages/DiaryHome/DiaryHome.js [hunk 1]
```diff
@@ -12,39 +17,74 @@ import {
...
+    const [diaryData, setDiaryData] = useState([]);
+
+    const [open, setOpen] = useState(false); ///using to handle dialog state
+
+    const [color, setColor] = useState({///using to handle form border colors(change to red when errors)
+        title: "#3498db",
+        description: "#3498db"
+    });
+
+    const [formData, setData] = useState({///using to handle form data
```
Comment (Dulajdeshan) [inline src/pages/DiaryHome/DiaryHome.js]: Setter should be `setFormData`

File: src/pages/DiaryHome/DiaryHome.js [hunk 2]
```diff
@@ -12,39 +17,74 @@ import {
...
+        title: "",
+        description: ""
+    });
+
+    const [loading, isLoading] = useState(true)
+
+    ///Use-effect using
+    useEffect(() => {
```
Comment (Dulajdeshan) [inline src/pages/DiaryHome/DiaryHome.js:55]: This is not required in this task. And this is not the way how it is done. This task is only for understanding how firebase works and configuring firestore in to project.

**Review by Dulajdeshan — CHANGES_REQUESTED** [review]
Go through the comments

**Review by udaraJ08 — COMMENTED** [review]
[ITP-135] fixed the setData issue with changing into setFormData

**Review by Dulajdeshan — APPROVED** [review]

**Pull request merged**